bg_atlasgen/atlas_scripts/princeton_mouse.py

```diff
@@ -0,0 +1,232 @@
+__version__ = "0.1"
```

Comment:
This should just be `1`. This is the minor version of the atlas, the major version is set by the api version. So this will end up being `v 1.1`

Reply:
Could we leave it as `0` in that case? Or is the standard to release atlases as v1.1?

Reply:
`1.0` is fine (and is what the first versions of atlases tend to be).